mlos_bench/mlos_bench/service/config_persistence.py

```diff
@@ -45,7 +48,10 @@ def __init__(self, config: Optional[Dict[str, Any]] = None,
         """
         super().__init__(config, parent)
         self._config_path = self.config.get("config_path", [])
-        self._config_loader_service = self
+        self.config_loader_service = self
```

Comment:
We make this public so that we can access it in the tests and via `Environment` classes rather than having to store separate pointers to it.